spec/helpers/http_mock/template.lua

```diff
@@ -43,7 +43,7 @@ http {
 
     function assert(truthy, err) -- luacheck: ignore
       if truthy then
-        return truthy
+        return original_assert(truthy, err)
```

Comment:
I don't think it's necessary. Could you explain the motivation?

Reply:
If the first argument of assert is true, it will return all arguments that it accepted. 
Returning `truthy` will change the behavior and thus changing all codes that relies on this default behavior